frontend/src/photoView/photoView.js

```diff
@@ -348,163 +370,211 @@ export class PhotoView extends React.Component {
-                            }
+                                    {side[0].toUpperCase() + side.slice(1)} Side
+                                </button>
+                            ))}
                         </div>
                     </div>
+                    <div className='image-info col-12 col-lg-6'>
+                        <h5>Photographer</h5>
+                        <p>
+                            {photographerName || 'Unknown'}
+                            {
+                                photographerNumber
+                                    ? <span>
+                                        {' (Number: '}
+                                        <a href={`/photographer/${photographerNumber}/`}>
+                                            {photographerNumber}
+                                        </a>
+                                    )
+                                    </span>
+                                    : ' (Number: Unknown)'
+                            }
+                        </p>
+                        <h5 className="caption">Photographer Caption</h5>
+                        <p>{photographerCaption || 'None'}</p>
+
+                        <h5>Visual Analysis</h5>
+                        <div className="row">
+                            <div className="col-6">
+                                {(this.state.displaySide === 'cleaned')
+                                    ? <select
+                                        id="toggleSelect"
+                                        className="custom-select"
+                                        onChange={this.toggleStatus}
+                                        value={this.state.view}
+                                    >
+                                        <option value="0">None selected</option>
+                                        <option value="1">Perspective Lines</option>
+                                        <option value="2">Foreground Mask</option>
+                                        <option value="3">YOLO Model</option>
+                                    </select>
+                                    : <p>Not available</p>
+                                }
+                                {(this.state.view === 3 && this.state.displaySide === 'cleaned')
+                                    ? <p className={'px-3 my-0'}>
+                                        <i>Hover over the boxes to see the name of the object.</i>
+                                    </p>
+                                    : <span></span>
+                                }
+                            </div>
+                        </div>
 
-                    {analyses.map((analysisResult, index) => {
-                        const analysisConfig = ANALYSIS_CONFIGS[analysisResult.name];
-                        const parsedValue = JSON.parse(analysisResult.result);
+                        {analyses.map((analysisResult, index) => {
```

Comment:
Not for now but: we really should factor this whole chunk out of the render function!